src/database.cpp

```
@@ -129,142 +180,242 @@ static bool executeQuery(tfs::detail::Mysql_ptr& handle, std::string_view query,
+			while (!s.empty() && (s.front() == ' ' || s.front() == '\t' || s.front() == '\n' || s.front() == '\r')) {
+				s.remove_prefix(1);
+			}
+			return s;
+		};
+		q = trimLeft(q);
+		constexpr auto cmdSize = std::min(sizeof("START TRANSACTION"), sizeof("ROLLBACK"));
+		char upper[cmdSize];
+		for (size_t i = 0; i < cmdSize && i < q.size(); ++i) {
+			upper[i] = static_cast<char>(std::toupper(static_cast<unsigned char>(q[i])));
+		}
+		std::string_view cmd{upper, cmdSize > q.size() ? q.size() : cmdSize - 1};
```

Cppcheck (2.20.0)

[warning] Uninitialized variable

(uninitvar)